src/Command/AddNewStudentsToSchoolCommand.php

```diff
@@ -99,24 +99,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
             );
         }
         $output->writeln("<info>Searching for new students to add to " . $school->getTitle() . ".</info>");
-        
+
         $students = $this->directory->findByLdapFilter($filter);
-        
+
         if (!$students) {
             $output->writeln("<error>{$filter} returned no results.</error>");
-            return;
+            return 0;
```

Comment:
i'm not sure if this really constitutes and error. and if so, should we return `1` instead then?

Reply:
No, I think it's just regular life. `0`.